web/main/models.py

```diff
@@ -2112,7 +2112,7 @@ def type(self):
         else:
             return 'resource'
 
-    def export(self, include_annotations, file_type='docx', export_options=None, is_child=False, experimental=False, aws_lambda=False, docx_footnotes=False):
+    def export(self, include_annotations, file_type='docx', export_options=None, is_child=False, experimental=settings.FORCE_EXPERIMENTAL_EXPORT, aws_lambda=settings.FORCE_AWS_LAMBDA_EXPORT, docx_footnotes=settings.FORCE_DOCX_FOOTNOTES):
```

Comment:
If using them as default arguments this way isn't offensive, I can switch them all from `FORCE` to `DEFAULT`.

Reply:
Interesting! I am a little anxious about it because of https://stackoverflow.com/questions/11238514/why-is-djangos-settings-object-a-lazyobject, but I'm not sure if that's in play here.....

Reply:
Haha! Best to avoid that just in case.